### ISSUE TITLE
Missing migration

### ISSUE THREAD

**Issue by joke2k** [django-notifications/django-notifications#118]
When I run `python manage.py makemigrations` django1.9 creates new migration for notifications package:
```python
...
class Migration(migrations.Migration):

    dependencies = [
        ('notifications', '0004_auto_20150826_1508'),
    ]

    operations = [
        migrations.AlterField(
            model_name='notification',
            name='level',
            field=models.CharField(max_length=20, default='info', choices=[('success', 'success'), ('info', 'info'), ('warning', 'warning'), ('error', 'error')]),
        ),
    ]
```

whats wrong?

**Comment by adamn** [comment]
Same problem here.  It's an issue with the migrations in the PyPI package.

**Comment by yangyubo** [comment]
Should has been fixed in PR #132
